src/com/fedorowiat/dbservice/DataBaseService.java

```diff
@@ -32,83 +28,27 @@ public void registerAccount() {
                 statement.setString(4, "" + user.getPassword());
                 statement.setString(5, "" + user.getPermissions());
                 statement.executeUpdate();
-                System.out.println("Konto zostało pomyślnie utworzone.");
-            }
         } catch (SQLException e) {
             System.out.println(e.getMessage());
         }
     }
 
-    public boolean checkEmail(){
 
+    public String getEmail(String email) {
```

Comment:
To troche nie ma sensu. Metoda getEmail a w parametrze podaje sie email, to co ona robi?